cmd/brew-bundle.rb

```diff
@@ -71,6 +73,8 @@
     Bundle::Commands::Cleanup.run
   when "check"
     Bundle::Commands::Check.run
+  when "wrap"
+    Bundle::Commands::Wrap.run ARGV[1..-1]
```

Comment:
You may want to handle ARGV in a smarter way, so it allows certain flags passed to `brew-bundle` instead of whatever program is wrapped. For example, to support:
```bash
brew bundle wrap --file=/path/to/file command --command-flag
# OR
brew bundle wrap --verbose -- command --command-flag
```
You could change `ARGV.named[0]` in the case block to `ARGV.shift`, then 
```ruby
  args = []

  ARGV.each_with_index do |arg, i|
    if arg == "--"
      args = ARGV.slice!(i+1..-1)
      break
    elsif !arg.start_with?("-")
      args = ARGV.slice!(i..-1)
      break
    end
  end
```

So the rest of ARGV will be used in `brew bundle` while `args` will be used by the wrapper.